bug: RadioGroup "compareWith" not available in Angular Standalone Components

### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/docs/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v8.x

### Current Behavior

When using the [`compareWith` property](https://ionicframework.com/docs/api/radio#object-value-references) of IonRadioGroup in a Ionic Angular Standalone components I get the following error:

> NG8002: Can't bind to 'compareWith' since it isn't a known property of 'ion-radio-group'.

Quick look over the code it seems like it has only been forgotten to be added here:

https://github.com/ionic-team/ionic-framework/blob/7623ae4d16ac49da1f616aae7819e31b29fca59b/packages/angular/standalone/src/directives/radio-group.ts#L19C7-L19C25

To test this I quickly added the property to `nodes_modules/@ionic/angular/standalone/directives/radio-group.d.ts` and everything worked as expected.


### Expected Behavior

The [`compareWith`](https://ionicframework.com/docs/api/radio-group#comparewith) property should be available in standalone components. 

### Steps to Reproduce

1. Create a standalone angular component
2. import IonRadioGroup
3. create a template like `<ion-radio-group [(ngModel)]="foomodel" [compareWith]="compareWith">`
4. Get compiler error.

### Code Reproduction URL

https://stackblitz.com/edit/angular-pe1kyd?file=src%2Fapp%2Fexample.component.html

### Ionic Info

node@b085e8bce2dc:~/app/sen2cube-mobile$ ionic info
[WARN] Error loading @capacitor/ios package.json: Error: Cannot find module '@capacitor/ios/package.json'
       
       Require stack:
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/lib/project/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/lib/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/bin/ionic
[WARN] Error loading @capacitor/android package.json: Error: Cannot find module '@capacitor/android/package.json'
       
       Require stack:
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/lib/project/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/lib/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/index.js
       - /home/node/.npm-global/lib/node_modules/@ionic/cli/bin/ionic

Ionic:

   Ionic CLI                     : 7.2.0 (/home/node/.npm-global/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 8.0.1
   @angular-devkit/build-angular : 17.3.6
   @angular-devkit/schematics    : 17.3.6
   @angular/cli                  : 17.3.6
   @ionic/angular-toolkit        : 11.0.1

Capacitor:

   Capacitor CLI      : 6.0.0
   @capacitor/android : not installed
   @capacitor/core    : 6.0.0
   @capacitor/ios     : not installed

Utility:

   cordova-res : 0.15.4
   native-run  : 2.0.1

System:

   NodeJS : v20.17.0 (/usr/local/bin/node)
   npm    : 10.8.2
   OS     : Linux 6.6

### Additional Information

_No response_

--- a/packages/angular/standalone/src/directives/radio-group.ts
+++ b/packages/angular/standalone/src/directives/radio-group.ts
@@ -16,7 +16,7 @@ import { defineCustomElement } from '@ionic/core/components/ion-radio-group.js';
 
 import { ProxyCmp, proxyOutputs } from './angular-component-lib/utils';
 
-const RADIO_GROUP_INPUTS = ['allowEmptySelection', 'name', 'value'];
+const RADIO_GROUP_INPUTS = ['allowEmptySelection', 'compareWith', 'name', 'value'];
 
 /**
  * Pulling the provider into an object and using PURE  works

